rust_src/src/symbols.rs

```diff
@@ -23,6 +23,10 @@ use crate::{
 pub type LispSymbolRef = ExternalPtr<Lisp_Symbol>;
 
 impl LispSymbolRef {
```

Comment:
Does it make more sense to drop this style and only use `From` traits?

Reply:
You mean `as_lisp_obj` ?

Reply:
Yes.

Reply:
Makes sense to me. This method has always looked a little strange.